NobelLaureateDetails/Pages/Index.cshtml.cs

```diff
@@ -29,17 +30,27 @@ public JsonResult OnGet()
                 }
 
                 foreach (long id in filteredIds) {
-                    string laureateDetails = webClient.DownloadString("http://api.nobelprize.org/v1/laureate.json?id="+id);
+                    //string laureateDetails = webClient.DownloadString("http://api.nobelprize.org/v1/laureate.json?id="+id);
+                    string laureateDetails = GetData("http://api.nobelprize.org/v1/laureate.json?id=" + id);
+                    LaureateDetails laureateDetailsList = LaureateDetails.FromJson(laureateDetails);
```

Comment:
This is a better name, though I'd drop the word "list" and just leave it as 'laureateDetails" The plural form often indicates a collection type.